src/interfaces.js

```diff
@@ -83,9 +83,15 @@ export function mapTagOperations({spec, cb = nullFn, defaultTag = 'default'}) {
       // Id already exists?
       if (typeof tagObj[id] !== 'undefined') {
         // Bump counter ( for this operationId )
-        operationIdCounter[id] = (operationIdCounter[id] || 0) + 1
+        let originalCounterValue = (operationIdCounter[id] || 1)
```

Comment:
@buunguyen Ah, I understand you now. I just pushed a fix for this case, and expanded the test case to match: https://github.com/swagger-api/swagger-js/pull/1048/commits/8dfc734f3641b821dfc410f19678b7ad01657666